src/js/views/patients/sidebar/filters/filters-sidebar_views.js

```diff
@@ -91,18 +91,35 @@ const FiltersCountView = View.extend({
   },
 });
 
+const ClearFiltersView = View.extend({
+  template: hbs`
+    <button class="filters-sidebar__clear-filters u-margin--r-16 js-clear-filters" {{#if isDisabled}}disabled{{/if}}>
```

Comment:
If there's no filters being applied, add the `disabled` attribute to the button.